src/engine/idTable/IdTableRow.h

```diff
@@ -266,9 +268,11 @@ class RowReferenceImpl {
 
     // Equality comparison. Works between two `RowReference`s, but also between
     // a `RowReference` and a `Row` if the number of columns match.
-    CPP_template(typename U)(requires(numStaticColumns ==
-                                      U::numStaticColumns)) bool
-    operator==(const U& other) const {
+    // Note: QCC 8.3 doesn't have access to `U::numStaticColumns`.
```

Comment:
Do you really mean QCC? And if yes, why do I have the feeling this is the first time this is mentioned in the codebase?